codegen/src/main/java/com/paypal/selion/plugins/CodeGeneratorMojo.java

```diff
@@ -93,6 +89,13 @@
     private List<String> androidCustomElements;
 
     /**
+     * List of "mobile" custom elements to be included during code generation.
+     *
+     * @parameter expression="${selion-code-generator.mobileCustomElements}"
+     */
+    private List<String> mobileCustomElements;
```

Comment:
@alizelzele fyi, you will be impacted here in a minor way by #287 if/when it is merged.
 We can fix when before we merge your PR.